Libraries/CO.CDP.Organisation.WebApiClient.Tests/OrganisationClientIntegrationTest.cs

```diff
@@ -39,6 +39,8 @@ public async Task ItTalksToTheOrganisationApi()
     };
         var types = new List<int> { 1 };
         var newOrganisation = new NewOrganisation(
+            // @TODO: pass the actual person id
```

Comment:
Forgive me this TODO, but we need to find a way to run these integration tests. Something to discuss on the next dev meet.